text/morph.js

```diff
@@ -188,6 +188,9 @@ export class Text extends Morph {
       },
 
       padding: {
+        spec: {
```

Comment:
Is it necessary to have a sub-field "spec"? Why not add type/values directly to property?